src/css/body-content.scss

```diff
@@ -0,0 +1,65 @@
+@import "colors";
+.box{
+    margin-bottom: 30px;
+    margin-top: 22px;
+    float: left;
+    display: flex;
+    flex-direction: column;
+
+    &__title{
+        font-size: 22px;
+        line-height: 1.3em;
+        color: $red;
+        text-align: left;
+        margin-bottom: 6px;
+        padding-bottom: 5px;
+        font-weight: 600;
+    } 
+    &__link{
+        &__photo{ 
+            width: 90%;
+            background-color: $white;
+            border: 1px solid $grey84;
+            position: relative;
+            float: left;
+            margin: 16px 13px 8px 0;
+            padding: 8px;
+
+        }
+    }
+
+    &__button{
+        text-decoration: underline;
+
+        &__title{
+            position: relative;
+            padding: 14px;
+            background-color: $black;
+            display: inline-block;
+            color: $white;
+            margin-top: 14px;
+            text-align: center;
+            text-decoration: none;
+            font: 14px/100% 'Open Sans', sans-serif;
+            border-radius: 5px;
+            border-radius: 5px;
+            padding: 12px 18px;
+            font-size: 1em;
+        }
+    
+    }
+}
+
+@media (min-width: 768px)
+{
+    .button__title{
```

Comment:
I know you do this because it's just one class, but keep on mind that you have to produce maintainable code